sklearn/preprocessing/_function_transformer.py

```diff
@@ -234,7 +236,8 @@ def inverse_transform(self, X):
 
         Parameters
         ----------
-        X : array-like, shape (n_samples, n_features)
+        X : {array-like, sparse-matrix, dataframe} of shape (n_samples, n_features) \
+        if validate=True else any object that ``func`` can handle
```

Comment:
`func` and `inverse_func` need not take the same input when we do not validate, hence we should specify

```suggestion
        if validate=True else any object that ``inverse_func`` can handle
```